src/app/services/search.service.ts

```diff
@@ -392,8 +406,39 @@ export class SearchService {
     }
     return this.client.post<fhirclient.FHIR.Bundle>(this.config.getServiceURL() + '/getClinicalTrial', query).pipe(
       map((bundle: fhirclient.FHIR.Bundle) => {
-        return new SearchResultsBundle(bundle, this.distService, zipCode);
+        return new SearchResultsBundle(bundle, this.distService, zipCode, this.config.getServiceURL());
       })
     );
   }
+  searchAllTrials(patientBundle: PatientBundle, offset?: number, count = 10): Observable<SearchResultsBundle[]> {
```

Comment:
I really don't like having the bulk of the code here be copied from `searchClinicalTrials`. I'm not sure it makes sense to keep the old method if it's going to basically do the same thing. Instead it probably makes sense to always take an array of URLs. If you only want to search one service, you just pass a single URL.